Moli/Core/Screens/Profile/BannerImageView.swift

```diff
@@ -3,23 +3,32 @@ import SwiftUI
 struct BannerImageView: View {
     var bannerImages: API.User.BannerThumbnailContainer?
     var isMe: Bool
+    var isEditProfileMode = false
 
     var imagePlaceholder: some View {
         ProgressView()
     }
 
     var body: some View {
-        HStack(spacing: 0) {
-            if let bannerImageURL = bannerImages?.M.url, let url = URL(string: bannerImageURL) {
-                AsyncImage(url: url) { image in
-                    image.resizable()
-                } placeholder: {
-                    imagePlaceholder
+        ZStack(alignment: .top) {
+            HStack(spacing: 0) {
+                if let bannerImageURL = bannerImages?.M.url, let url = URL(string: bannerImageURL) {
+                    AsyncImage(url: url) { image in
+                        image.resizable()
+                    } placeholder: {
+                        imagePlaceholder
+                    }
+                } else {
+                    Text(isMe ? "Choose an image" : "")
+                        .foregroundColor(.gray400)
+                        .padding(.top)
                 }
-            } else {
-                Text(isMe ? "Choose an image" : "")
-                    .foregroundColor(.gray400)
-                    .padding(.top)
+            }
+            if isEditProfileMode {
+                Text("Change cover photo")
+                    .foregroundColor(.white)
+                    .font(.paragraphSmall)
+                    .padding(.top, 20)
```

Comment:
:bomb: Hittade ett fall som du kan ha missat, om man tar bort sin bild så är både "Change cover photo" texten och "Choose an image" texten kvar

![image](https://user-images.githubusercontent.com/10032344/152518438-25145f2c-4497-494c-b73e-2dfba406e739.png)

Lyckades fixa genom att:

```suggestion
            if isEditProfileMode && bannerImages != nil {
                Text("Change cover photo")
                    .foregroundColor(.white)
                    .font(.paragraphSmall)
                    .padding(.top, 20)
```

Reply:
👑  Snyggt fångat!